src/test/kotlin/no/nav/syfo/melding/cronjob/UbesvartMeldingCronjobSpek.kt

```diff
@@ -59,7 +77,7 @@ class UbesvartMeldingCronjobSpek : Spek({
                         result.updated shouldBeEqualTo 1
                     }
 
-                    // TODO: Add check for publish to kafka
+                    verify(exactly = 1) { kafkaUbesvartMeldingProducer.sendUbesvartMelding(any(), any()) }
```

Comment:
Kunne også testet at kafka-objektet har forventede verdier i en eller flere av disse testene (ved å f.eks. mocke selve KafkaProducer-en og capture det som blir sendt, se MeldingApiSpek for eksempel)

Reply:
Ja, ser det er en del config da, for å få den funksjonaliteten 🤔 Man må vel override `cronjobModule` slik at man kan sende inn en mocket producer. Blir litt mer rotete prod-kode ut av det også, så det kommer an på om vi synes det er verdt det for å kunne teste eller ikke. Tanker?

Reply:
Du har rett, det var ikke så mye verre enn det 👍🏼 La til test på det nå 🙌🏼